sources/packages/backend/libs/services/src/notifications/notification/notification.model.ts

```diff
@@ -1,10 +1,15 @@
 import { NotificationMessageType } from "@sims/sims-db";
 import { NotificationEmailMessage } from "./gc-notify.model";
 
+interface NotificationMetadata {
+  disbursementId: number;
```

Comment:
Since `NotificationMetadata` is supposed to map any future metadata, `disbursementId` would probably needed to nullable.
Not a blocker right now I would recommend changing it.